test/ci/kokoro/config_generator.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
```

Comment:
Super small nit, 2018 -> 2019 since this is a new file.

Reply:
Fixed :)